CMakeLists.txt

```diff
@@ -536,6 +536,8 @@ set(CPPCHECK_ARGS
   # False positives on structured bindings with 2.5
   "--suppress=unassignedVariable"
   "--suppress=unusedVariable"
+  # TODO(laurynas): figure out and fix
+  "--suppress=throwInNoexceptFunction"
```

Comment:
@laurynas-biveinis Could this be related to the clang-specific oddness around TYPED_TEST(ARTCorrectnessTest, DISABLED_TooLongValue)?

Reply:
`TooLongValue` fixed. The cppcheck error no longer reproduces, removing this